packages/common/rollup-configs.js

```diff
@@ -1,11 +1,13 @@
 import { execSync } from 'child_process';
-/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-extraneous-dependencies -- avoid pulling Rollup dependencies into common package.json */
```

Comment:
This addresses @andrewballantyne's review comment https://github.com/openshift/dynamic-plugin-sdk/pull/23#discussion_r774043939